tests/unit/ValidationUtilsTest.js

```diff
@@ -1,6 +1,206 @@
 const ValidationUtils = require('../../src/libs/ValidationUtils');
+const moment = require('moment');
 
 describe('ValidationUtils', () => {
+    describe('isValidDate', () => {
+        test('isValidDate should return true for a valid date within the range', () => {
```

Comment:
The same suggestion for the all next tests. You already specified in `describe` which function do you test.
```suggestion
        test('should return true for a valid date within the range', () => {
```